Lib/cu2qu/__init__.py

```diff
@@ -177,9 +177,9 @@ def cubic_approx_spline(cubic, n, tolerance):
         if i == 1:
             q0, q1, q2 = spline[0], spline[1], (spline[1] + spline[2]) * .5
         elif i == n:
-            q0, q1, q2 = (spline[-3] + spline[-2]) * .5, spline[-2], spline[-1]
+            q0, q1, q2 = q2, spline[-2], spline[-1]
```

Comment:
I was thinking about reusing q2.  It's nice, but belongs to separate commit.  Don't worry separating it though.